README.md

```diff
@@ -5,15 +5,15 @@ This repository contains official Helm charts that are maintained by [Aiven](htt
 ## Adding the repository
 
 ```bash
-helm repo add aiven https://aiven-charts.github.io && helm repo update
+helm repo add aiven aiven https://aiven.github.io/aiven-charts && helm repo update
 ```
 
 ## Installing a chart from the repository
 
 ### Installing the aiven-operator-crds chart
```

Comment:
Can we link to the main official docs instead of having the instructions here? https://aiven.github.io/aiven-operator/docs/installation/helm/

I know the instructions are not there yet, but will be after merging [this one](https://github.com/aiven/aiven-operator/pull/158/files).

Reply:
done